documentation/features.asciidoc

```diff
@@ -1,7 +1,7 @@
 :toc:
 toc::[]
 
-= Features
+== Why do you need devonfw-ide?
```

Comment:
In the introduction, you have the same headline. Should they be combined?

Reply:
removed features.asciidoc since the content already exists in introduction.